SR2025/2024-11-13-check-in.md

```diff
@@ -0,0 +1,26 @@
+---
+to: SR2025 teams
+subject: SR2025 November Check-in
+---
+
+Hi {PrimaryGivenName},
+
+Just wanted to check in on how your team is doing?
+
+It would be great to hear what your team is currently working on and if there's
+anything you'd like a hand with.
+
+{% if not in discord %}
+We noticed that your team isn't present in Discord -- is this something that
+you'd be able to encourage them to join? Teams which are in Discord are able to
+get help much more easily from our volunteers and tend to do better overall.
+{% endif %}
+
+As a reminder, if you'd like to come to the [Horsham Tech Day][horsham-tech-day]
+please [let us know][tech-day-signup] this week.
+
+Thanks,\
```

Comment:
```suggestion
Thanks,
```
Unless it's there for a reason?

Reply:
In non-GitHub-flavoured markdown, the trailing `\` prevents the text being wrapped into a single paragraph.

So the difference in rendering is between:
```
Thanks, Peter
```
and
```
Thanks,
Peter
```

Reply:
Ah, thanks for the explanation!